docker/service/src/main/java/uk/nhs/adaptors/scr/consts/Numbers.java

```diff
@@ -0,0 +1,7 @@
+package uk.nhs.adaptors.scr.consts;
+
+public class Numbers {
+    public static final int ONE = 1;
+    public static final int TWO = 2;
+    public static final int THREE = 3;
```

Comment:
rename these to reflect classes they're used in 

Something like, CORRELATION_ID_FILTER_ORDER